ui/src/dashboards/components/GraphOptionsTimeAxis.js

```diff
@@ -1,29 +1,30 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-const VERTICAL = 'VERTICAL'
-const HORIZONTAL = 'HORIZONTAL'
-const GraphOptionsTimeAxis = ({TimeAxis, onToggleTimeAxis}) =>
+const GraphOptionsTimeAxis = ({verticalTimeAxis, onToggleVerticalTimeAxis}) =>
```

Comment:
this component could be converted to typescript

Reply:
I didn't do this.